README.md

```diff
@@ -657,6 +657,14 @@ Valid time units include `s`, `m`, and `h`, e.g. `1h`, `1m1s`.
 * `settings.ecs.reserved-memory`: The amount of memory, in MiB, reserved for critical system processes.
 * `settings.ecs.task-cleanup-wait`: Time to wait before the task's containers are removed after they are stopped.
 Valid time units are `s`, `m`, and `h`, e.g. `1h`, `1m1s`.
+* `settings.ecs.image-cleanup-wait`: Time to wait between image cleanup cycles
```

Comment:
Nit: missing period at the end of this sentence. With the next line being part of the same bullet, that will cause the two to run together.

Second lines should really be indented too, so they are clearly part of the same bullet item line. But at least for now, it looks like GitHub's markdown rendering handles this OK so the rendered output is correct.